docs/source/tutorial/package_a_project.md

```diff
@@ -145,6 +135,18 @@ main(
 
 This is equivalent to running `kedro run`, and you can provide all the parameters described by `kedro run --help`.
 
+To install the package from a `.whl` file, you need to have Python and `pip` on your machine, but do not need to have Kedro installed.
+
+```{note}
+Once the packaged project is installed, you will need to add:
+
+* a `conf` folder
+* a `data` folder if the pipeline loads/saves local data
+* a `logs` folder if the project uses logging.
+Alternatively, they can make use of the ``OmegaConfigLoader`` to run the configuration directly from the compressed .tar.gz configuration file by running
```

Comment:
```suggestion
* a `logs` folder if the project uses logging.

Alternatively, you can make use of the ``OmegaConfigLoader`` to run the configuration directly from the compressed .tar.gz configuration file by running
```

Fixing whitespace and replacing "they" by "you" (hope I understood this correctly)

Reply:
Thanks for noticing!